org.moonsoft.MoonPlayer.json

```
@@ -0,0 +1,195 @@
+{
+  "app-id": "org.moonsoft.MoonPlayer",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "moonplayer",
+  "rename-icon": "moonplayer",
+  "rename-desktop-file": "moonplayer.desktop",
+  "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=wayland",
+      "--device=all",
+      "--share=network",
+      "--socket=pulseaudio",
+      "--filesystem=host",
+      "--env=LC_NUMERIC=C"
+  ],
+  "modules": [
+    {
+      "name": "moonplayer",
+      "buildsystem": "qmake",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/coslyk/moonplayer/archive/2.1.1.tar.gz",
+          "sha256": "67c74681d12c10a4ecc4c2bad22006e22bb804b546ba470cac0408b1f02f1f58"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "sed -i -e 's/PKGCONFIG += python2 mpv/PKGCONFIG += qtermwidget5 python2 mpv/g' src/moonplayer.pro",
+            "sed -i -e 's|unix:!macx: INCLUDEPATH += /usr/include/qtermwidget5||g' src/moonplayer.pro",
+            "sed -i -e 's|unix:!macx: LIBS += -lqtermwidget5||g' src/moonplayer.pro",
+            "sed -i -e 's|/usr/|/app/|g' src/moonplayer.pro",
+            "sed -i -e 's|QT;||g' src/moonplayer.desktop"
+          ]
+        }
+      ],
+      "subdir": "src",
+      "modules": [
+        {
+          "name": "qtermwidget",
+          "cleanup": [ "/include", "/lib/pkgconfig", "/lib/cmake" ],
+          "buildsystem": "cmake",
```

```suggestion
          "buildsystem": "cmake-ninja",
```

(This may or may not work)

---

This works for me, thanks